installer/dev/windowsappruntime_definitions.h

```diff
@@ -0,0 +1,186 @@
+﻿  #pragma once
```

Comment:
new file?  was this supposed to be in the rename?  of this is a rename that github didn't recognize?  (non-blocking, just curious)

Reply:
nevermind, this is an unrecognized rename